ingester/src/wal/wal_sink.rs

```diff
@@ -140,14 +152,36 @@ mod tests {
     async fn test_append() {
         let dir = tempfile::tempdir().unwrap();
 
-        // Generate the test op that will be appended and read back
-        let op = make_write_op(
-            &PartitionKey::from("p1"),
+        const SECOND_TABLE_ID: TableId = TableId::new(45);
+        const SECOND_TABLE_NAME: &str = "banani";
+        // Generate a test op containing writes for multiple tables that will
+        // be appended and read back
+        let mut tables_by_name = lines_to_batches(
+            r#"bananas,region=Madrid temp=35 4242424242
+banani,region=Iceland temp=25 7676767676"#,
```

Comment:
[Cool trick](https://doc.rust-lang.org/reference/tokens.html#string-literals):

> when an unescaped U+005C character (\) occurs immediately before a line break, then the line break character(s), and all immediately following (U+0020), \t (U+0009), \n (U+000A) and \r (U+0000D) characters are ignored.

This means we can do this for readability:

```suggestion
            "bananas,region=Madrid temp=35 4242424242 \
             banani,region=Iceland temp=25 7676767676",
```

Without affecting the string value! (kinda, I added a space at the end of the first line which shouldn't affect parsing)

This only works for normal string literals ("") and not _raw_ string literals (r#""#):

https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=1340bb1edc0e667546b05167e0ad22d4

Reply:
Trés chic, merci 😎 

Much nicer to read.